modules/carto/src/layers/h3-tile-layer.js

```diff
@@ -0,0 +1,43 @@
+import {CompositeLayer} from '@deck.gl/core';
+import {H3HexagonLayer, TileLayer} from '@deck.gl/geo-layers';
+import H3Tileset2D from './h3-tileset-2d';
+
+const renderSubLayers = props => {
+  const {data} = props;
+  const {index} = props.tile;
+  if (!data || !data.length) return null;
+
+  return new H3HexagonLayer(props, {
+    centerHexagon: index,
```

Comment:
Would be great to have a defaultProp to avoid the user to write at CartoLayer:
`getHexagon: d => d.id`